sdk/Cargo.toml

```diff
@@ -60,6 +60,7 @@ solana-sdk-macro-frozen-abi = { path = "macro-frozen-abi", version = "1.3.0" }
 rustversion = "1.0.3"
 
 [dev-dependencies]
+curve25519-dalek = { version = "2", default-features = false }
```

Comment:
Why `default_features = false`?  Cut down on bloat?

Reply:
I just snagged the dep line out of `ed25519-dalek`s `Cargo.toml`. Happy to drop it if you like